### PR TITLE
docs(samples): Update metadata and arguments

### PR DESCRIPTION
**Pull request by averikitsch**
Fixes #<issue_number_goes_here> (it's a good idea to open an issue first for discussion)

- [ ] Tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

### CODE REVIEW

**Comment by averikitsch** [comment]
Fixes #229

File: samples/createHttpTask.js [hunk 1]
```diff
@@ -18,17 +18,18 @@
 // sample-metadata:
 //   title: Cloud Tasks Create HTTP Target
 //   description: Create Cloud Tasks with a HTTP Target
+//   usage: node createHttpTask.js
```
Comment (JustinBeckwith) [inline samples/createHttpTask.js]: The idea here is to call out which arguments the user can pass to the program, so they know the usage info.  Can we make this something like:

```js
//    usage: node createHttpTask.js <projectId> <queueName> <location> <url> <payload> <delayInSeconds> 
```

File: samples/createHttpTaskWithToken.js [hunk 2]
```diff
@@ -18,18 +18,19 @@
 // sample-metadata:
 //   title: Cloud Tasks Create HTTP Target with Token
 //   description: Create Cloud Tasks with a HTTP Target with Token
+//   usage: node createHttpTaskWithToken.js
```
Comment (JustinBeckwith) [inline samples/createHttpTaskWithToken.js]: same deal here

File: samples/createHttpTask.js (hunk 1 above)
Comment (averikitsch) [reply samples/createHttpTask.js]: Now that I've add the default arguments, do you think this will confuse the user?

**Pull request merged**